### DISCUSSION TITLE
[@types/three] Missing definition for THREE.ColorManagement.legacyMode

### DISCUSSION THREAD

**Discussion by fulopkovacs** [DefinitelyTyped/DefinitelyTyped#61992] [Issues with a @types package] [answered]
# Description

I  have discovered a few issues regarding the `ColorManagement` module:
- `ColorManagement` has a `legacyMode` property that is missing from the type definitions. Here's the property in the latest release of `three` (`three@0.143.0`): [link](https://github.com/mrdoob/three.js/blob/e22cb060cc91283d250e704f886528e1be593f45/src/math/ColorManagement.js#L24).
- `ColorManagement` is not exported properly. See how it's exported in  [`threejs`](https://github.com/mrdoob/three.js/blob/e22cb060cc91283d250e704f886528e1be593f45/src/Three.js#L131), but not in [`@types/three`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/380e22ab4c1afbf96a35b1dc879b8acb04ecddf8/types/three/src/Three.d.ts#L135).
  - Currently this code example is valid according to the latest `threejs` release, but not according to the type definitions:
  ```typescript
  import * as THREE from 'three';
  console.log(THREE.ColorManagement);
  ```

I'd be happy to make a PR if that helps. :relaxed:

[a comment by an automated account is not shown]

**Comment by joshuaellis** [comment, marked as answer]
Please open this issue in the three-ts-types repo.

**Reply by fulopkovacs** [reply]
Done: https://github.com/three-types/three-ts-types/issues/250#issue-1355390632! :sparkles:

**Reply by joshuaellis** [reply]
Thanks, I'll review soon!!
